pkg/catalog/catalog_test.go

```diff
@@ -178,13 +178,17 @@ func TestSliceSupportedTags(t *testing.T) {
 			},
 		})
 
-	img := CatalogImage{
-		Name:      "nodejs",
-		Namespace: "openshift",
-		NonHiddenTags: []string{
-			"10", "8", "6", "latest",
+	img := ComponentType{
+		ObjectMeta: metav1.ObjectMeta{
```

Comment:
no type meta on this?

Reply:
No need for the test, but I could add it :)

Reply:
Ok